pbdf/description.xml

```diff
@@ -0,0 +1,15 @@
+<Issuer version="4">
+	<ID>pbdf</ID>
+	<ShortName>
+		<en>Demo PbD.f</en>
```

Comment:
Is this dot actually part of the foundation's name?

Reply:
I simply copied it from the `pbdf-schememanager`. It is there too. No idea why this was done.